awswrangler/s3/_write_parquet.py

```diff
@@ -64,6 +64,8 @@ def _new_writer(
         pyarrow_additional_kwargs["use_dictionary"] = True
     if not pyarrow_additional_kwargs.get("write_statistics"):
         pyarrow_additional_kwargs["write_statistics"] = True
+    if schema:
```

Comment:
Wouldn't that always give priority to the schema we compute? As the issue mentioned, we always [infer](https://github.com/aws/aws-sdk-pandas/blob/main/awswrangler/s3/_write.py#L328C20-L328C20) the schema, so that condition would always evaluate to True

Reply:
Ah, you're absolutely right - reversed the schemas. Thanks!